script/rules/YieldNestRules.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.24;
+
+import {IVault, IValidator} from "src/interface/IVault.sol";
+import {SafeRules} from "./SafeRules.sol";
+
+library YieldNestRules {
```

Comment:
no ynETH and yLSDe on bsc let's remove it

Reply:
removed